backend/src/akvo/lumen/update.clj

```diff
@@ -81,12 +78,13 @@
     (let [table-name (util/gen-table-name "ds")
           imported-table-name (util/gen-table-name "imported")
           imported-columns (vec
-                            (:columns (imported-dataset-columns-by-dataset-id conn
-                                                                              {:dataset-id dataset-id})))
+                            (:columns (imported-dataset-columns-by-dataset-id
+                                       conn {:dataset-id dataset-id})))
           {:keys [transformations] :as dataset-version} (latest-dataset-version-by-dataset-id
                                                          conn {:dataset-id dataset-id})]
       (with-open [importer (import/dataset-importer (get data-source-spec "source") config)]
-        (let [columns (import/columns importer)]
+        (let [columns (map #(update-in % [:title] string/trim)
```

Comment:
Can this not just be:

```clojure
(update % :title string/trim)
```

Don't really need `(update-in)` since it's just a single toplevel key

Reply:
👍